packages/manager/apps/hub/src/dashboard/welcome/template.html

```diff
@@ -1,6 +1,9 @@
 <h1
-    class="oui-heading_2"
+    class="oui-heading_2 d-inline-block"
     data-translate="manager_hub_dashboard_welcome"
     data-translate-values="{ name: $ctrl.name }"
     data-navi-id="hub-welcome-title"
 ></h1>
+<ovh-manager-trusted-nic-label
+    data-ng-if="$ctrl.isTrusted"
```

Comment:
I'm wondering if the `ng-if` could directly be managed by component as at first guess `isTrusted ` will have the same value across all usage

Reply:
It could be done. But the idea was to keep the component simple... Secondly, it sounds a bit strange, when you use a component and the component hides itself...
But anyways, I prefer not to change it at this stage